README.md

```diff
@@ -1,29 +1,86 @@
 # FathomFox
 
-A set of tools for developing [Fathom](http://mozilla.github.io/fathom/) rulesets within Firefox
+A set of tools for developing [Fathom](http://mozilla.github.io/fathom/)
+rulesets within Firefox
+
+## Quick Start
+
+_Getting started with the "overlay" ruleset:_
+
+1. Fork, clone, and install the
+   [Fathon-Trainees](https://github.com/mozilla/fathom-trainees/) repository as per
+   its instructions
+2. Clone the [Fathom-Fox](https://github.com/mozilla/fathom-fox) repository
+3. Install dependencies with `yarn`, build with `yarn build`, then run `yarn browser`
+   to launch Firefox
+4. Using `about:debugging` install your `fathom-trainess` extension from its
+   `addon/manifest.json` file
+5. Navigate to a web site that has a modal dialog with a background overlay
+6. Using Firefox's developer tools inspector select the full-page overlay element
+7. Switch to the `Fathom` developer tools tab, enter `overlay` in the label field,
```

Comment:
Elsewhere I haven't been using monospace formatting for button names and such, just caps.